Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml

```diff
@@ -0,0 +1,1242 @@
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "16"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 1920
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "16":
+    id: "16"
+    taskid: 3bf08ec5-416b-47fd-89db-d0e5c2042a40
+    type: condition
+    task:
+      id: 3bf08ec5-416b-47fd-89db-d0e5c2042a40
+      version: -1
+      name: Any compromised hosts found?
+      description: |
+        Checks whether there was a compromised hosts found.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "19"
+      "yes":
+      - "17"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: Endpoint
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 2060
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "17":
+    id: "17"
+    taskid: 14bb0783-da07-4cd4-88a8-cb0cf5831974
+    type: condition
+    task:
+      id: 14bb0783-da07-4cd4-88a8-cb0cf5831974
+      version: -1
+      name: Should isolate automatically compromised hosts?
+      description: |
+        Checks whether files associated with the incident can be blocked automatically using various EDR integrations.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "20"
+      "yes":
+      - "18"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.IsolateEndpointAutomatically
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    view: |-
+      {
+        "position": {
+          "x": 1360,
+          "y": 2230
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "18":
+    id: "18"
+    taskid: 9d270ed1-773b-43a0-8bab-3a8821895516
+    type: playbook
+    task:
+      id: 9d270ed1-773b-43a0-8bab-3a8821895516
+      version: -1
+      name: Isolate Endpoint - Generic
+      playbookName: Isolate Endpoint - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      EndpointId: {}
+      Hostname:
+        complex:
+          root: Endpoint
+          accessor: HostName
+      IP:
```

Comment:
Need to make sure your not providing external IP addresses to block
Consider using something like IsIPInRanges with a playbook input that contains the relevant ranges
The Splunk Indicator Hunting can be used as a reference.